src/ui/home/components/claimDetails.tsx

```diff
@@ -36,30 +31,16 @@ const styles = StyleSheet.create<ClaimDetailsStyles>({
     marginBottom: viewHeight / 20,
   },
   buttonContainer: {
```

Comment:
`buttonContainer: Buttons.buttonStandardContainer` should do the same, since we are spreading without overwriting. Unless we want to keep it like this for consistency.